### ISSUE TITLE
TypeError: unexpected keyword argument 'socknum'

### ISSUE THREAD

**Issue by kyleplo** [adafruit/Adafruit_CircuitPython_ESP32SPI#103]
When I run the server example, I get this error.

I am running this on an Adafruit PyPortal

**Comment by jerryneedell** [comment]
Are you using the latest versions of the libraries?  I just ran it on my PyPortal and it works normally.

**Comment by kyleplo** [comment]
No, I just updated. Let me try again

**Comment by kyleplo** [comment]
Now it works!
